Revit_Core_Engine/Compute/Warnings.cs

```diff
@@ -514,6 +514,26 @@ internal static void TransformNotImplementedWarning(this Element element)
         }
 
         /***************************************************/
+
+        internal static void ElementOwnedByOtherUserWarning(Element element)
+        {
+            BH.Engine.Base.Compute.RecordWarning($"Revit object could not be updated or modified due to it's CheckoutStatus. Revit ElementId: {element.Id} is owned by another user.");
+        }
+
+        /***************************************************/
+
+        internal static void ElementOwnedByCurrentUserWarning(Element element)
```

Comment:
I do not find it being a warning - a note at best. But would rather not record it as event altogether, assuming it is the default case.